core/Container/ContainerFactory.php

```diff
@@ -82,4 +82,16 @@ private function addEnvironmentConfig(ContainerBuilder $builder)
 
         $builder->addDefinitions($file);
     }
+
+    private function addPluginConfigs(ContainerBuilder $builder)
+    {
+        $plugins = Config::getInstance()->Plugins['Plugins'];
```

Comment:
Can we load plugins before that or so? We should not really include configs of not activated plugins. Also in tracker mode we have much less plugins that need to be loaded/activated etc.